addon/lib/user.js

```diff
@@ -64,3 +68,129 @@ function makeUuid() {
 exports.getUserInfo = function () {
   return ss.storage.userInfo;
 };
+
+exports.updateProfileInfo = function (profile) {
```

Comment:
I guess `ss.storage.profileInfo` is info taken from your FxA, but `ss.storage.userInfo` is actually device info.  Maybe the latter should be renamed to clarify?  As it is even I get confused about which is which.